test/util/utils.go

```diff
@@ -90,6 +90,41 @@ func MakePG(name, namespace string, min int32, creationTime *time.Time, minResou
 	return pg
 }
 
+func UpdatePGStatus(pg *v1alpha1.PodGroup, phase v1alpha1.PodGroupPhase, occupiedBy string, scheduled int32, running int32, succeeded int32,
+	failed int32) *v1alpha1.PodGroup {
+	pg.Status = v1alpha1.PodGroupStatus{
+		Phase:      phase,
+		OccupiedBy: occupiedBy,
+		Scheduled:  scheduled,
+		Running:    running,
+		Succeeded:  succeeded,
+		Failed:     failed,
+	}
+	return pg
+}
+
+func PGStatusEquals(a, b v1alpha1.PodGroupStatus) bool {
```

Comment:
you can use https://pkg.go.dev/github.com/google/go-cmp/cmp#Equal `cmp.Equal` and do not implement struct matching yourself. This package is already used in a project.

Reply:
Thanks! Fixed it